training_program/tests/tests_views.py

```diff
@@ -0,0 +1,195 @@
+'''Unit tests for training_program views.'''
+from django.urls import reverse
+from model_mommy import mommy
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+import training_program.models
+import auth.models
+
+
+class TestProgramCategoryViewSet(APITestCase):
+    '''Unit tests for ProgramCategory view.'''
+    def test_create_programcategory(self):
+        '''ProgramCategory should be created by POST request.'''
+        url = reverse('programcategory-list')
+        name = 'programcategory'
+        data = {'name': name}
+
+        response = self.client.post(url, data, format='json')
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertEqual(training_program.models.ProgramCategory.
```

Comment:
https://github.com/DLUT-SIE/TMSFTT-BE/blob/24e4155fe4aa1002811d9fefe8a34f04192526e2/training_program/tests/tests_views.py#L22-L23
You can use:
```
self.assertEqual(
    training_program.models.ProgramCategory.objects.count(), 1)
```
or use a parenthesis to wrap your chain calls, like:
```
self.assertEqual((training_program.models.ProgramCategory
                 .objects.count()), 1)
```
Note that we move the dot to the next line and add a `()` to wrap the whole chain call.

IMHO, the former is a better practice, unless your code is too long.